backend/tests/e2e/graph-info/graph-info.service.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { Neo4jService } from 'nest-neo4j/dist';
+import { GraphInfoService } from '../../../src/graph-info/graph-info.service';
+import { KmapNeo4jModule } from '../../../src/config/neo4j/KmapNeo4jModule';
+import { AppModule } from '../../../src/app.module';
+import { edgeInfo, nodeInfo } from '../../fixtures/nodeInfo/GraphInfoDb';
+
+/*
+Tests graph-info.spec.ts
+Tests parseNeo4jEntityInfo indirectly
+ */
+describe('GraphInfoService', () => {
```

Comment:
since you are not mocking your Neo4jService this is an e2e-test right? if yes, add an "e2e" in the filename